### PR TITLE
Add product attributes

### PR DESCRIPTION
**Pull request by tomek-lochmanczyk-rspective**

### CODE REVIEW

File: src/import/product-import.service.ts [hunk 1]
```diff
@@ -76,6 +78,14 @@ export class ProductImportService {
...
 
   private async productImport(period?: number) {
+    const meatdataSchemas = await this.voucherifyClient
+      .getClient()
+      .metadataSchemas.list();
+    const metadataSchema = meatdataSchemas.schemas.find(
+      (schema) => schema.related_object === 'product',
+    );
+    const metadataSchemaProperties = Object.keys(metadataSchema.properties);
+
```
Comment (jkaliszuk) [inline src/import/product-import.service.ts:88]: 3x the same logic with different schema.related_object variable. Consider wrapping it in the same method e.g in VoucherifyConnectorService.

**Review by jkaliszuk — CHANGES_REQUESTED** [review]

**Review by p-zielinski — APPROVED** [review]
✔ Customers successfully migrated

File: src/import/customer-import.service.ts [hunk 2]
```diff
@@ -58,6 +60,13 @@ export class CustomerImportService {
   }
 
   private async customerImport(period?: number) {
+    const meatdataSchemas = await this.voucherifyClient
+      .getClient()
+      .metadataSchemas.list();
+    const metadataSchema = meatdataSchemas.schemas.find(
+      (schema) => schema.related_object === 'customer',
+    );
+    const metadataSchemaProperties = Object.keys(metadataSchema.properties);
```
Comment (p-zielinski) [inline src/import/customer-import.service.ts]: const metadataSchemaProperties = Object.keys(metadataSchema?.properties ?? {});

**Review by p-zielinski — CHANGES_REQUESTED** [review]
Unlikely, but I think it may cause error

File: src/import/customer-import.service.ts (hunk 2 above)
Comment (p-zielinski) [reply src/import/customer-import.service.ts]: U can throw an error if `metadataSchema.properties` return `undefined` as well.

**Review by p-zielinski — COMMENTED** [review]
no further changes requested

**Review by jkaliszuk — APPROVED** [review]

**Pull request closed without merging**